plugins/LilyPad/LilyPad.cpp

```diff
@@ -1195,15 +1188,15 @@ u8 CALLBACK PADpoll(u8 value) {
 				for (i = 0; i<8; i++) {
 					b2 -= (sum->buttons[i+4] > 0) << i;
 				}
+				for (i = 4; i<8; i++) {
+					b1 -= (sum->buttons[i+8] > 0) << i;
+				}
+				
```

Comment:
The tab characters don't really serve a purpose since there's no code on the line?